core/templates/pages/exploration-editor-page/exploration-editor-page.component.html

```diff
@@ -80,7 +80,7 @@ <h1>{{ getNavbarText() }}</h1>
 
       <li [ngClass]="{'navbar-tab-active': getActiveTabName() === 'settings', 'oppia-disabled-tab': !connectedToInternet}" class="nav-item icon nav-list-item e2e-test-settings-tab"
           (click)="!connectedToInternet || selectSettingsTab()" [ngbTooltip]="'Settings does not work when offline.'" [disableTooltip]="connectedToInternet" placement="bottom">
-        <a class="nav-link  navbar-tab"
+        <a class="nav-link  navbar-tab .e2e-test-settings-tab-icon"
```

Comment:
I think there should be 1 whitespace char instead of 2.

Reply:
Sorry, It's redundant, removed it.